libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -209,6 +209,23 @@ AP_GPS_ERB::_parse_gps(void)
         state.speed_accuracy = _buffer.vel.speed_accuracy * 0.01f;
         _new_speed = true;
         break;
+    case MSG_RTK:
+        Debug("Message RTK");
+        state.rtk_baseline_coords_type = 1;
```

Comment:
An enum would be a bonus here for clarity, but probably not worth holding up on merging for it.

Reply:
I added the enum RTK_BASELINE_COORDINATE_SYSTEM_NED in the mavlink https://github.com/ArduPilot/mavlink/commit/dbede4f34c70de7ee53ccc0164b53b981d7032ea
But I can not find it in any header file :(
Where is it ?

Reply:
Got it, and added it.